model/workplane.py

```diff
@@ -97,9 +97,13 @@ def draw(self, context):
 
     def draw_id_face(self, context, shader):
         """Draw only the face of the workplane to the selection buffer"""
-        # Selectability check removed, handled by caller (draw_selection_buffer)
-        # if not self.is_selectable(context):
-        #     return
+        # Check if workplane should be shown in selection buffer when inside a sketch
+        active_sketch = context.scene.sketcher.active_sketch
+        if active_sketch and active_sketch.wp != self:
```

Comment:
The workplane is a 3d element so it should never be drawn when a sketch is active

Reply:
It's not, I think. I split those out into two separate calls but didn't update the comments... Can you see it in the selection buffer?

Reply:
Yes i still see the black square in the selection buffer when a sketch is active

Reply:
The latest push should fix that. I made the logic a bit more explicit. But I still need to go back and validate that overlapping selections work and hoist a few more constants.